### PR TITLE
Ready to merge: add term: 'gastric intestinal metaplasia'

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
@sabrinatoro could you review this?

There is another metaplasia term in Mondo.

But this is a finding in NCIT

Also, I wonder if this NCIT xref would be more appropriate:
https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C3956

(vs https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C8360&lang=en&viewMode=All&siblings=false)

close https://github.com/monarch-initiative/mondo/issues/5594

### CODE REVIEW

**Comment by sabrinatoro** [comment]
I agree that http://purl.obolibrary.org/obo/NCIT_C3956 would be more appropriate x-ref. However, this term is, as you mentioned, in the "finding" branch of NCIT, and we are excluding these as there are not diseases. So the NCIT cannot be an equivalent term. I suggest using the source "MONDO:relatedTo" (=same by in a different hierarchy) 

The main question is: should this term be in Mondo? it looks like it might need to be in HPO instead as I am not sure that it is an actual disease. This part probably requires expert input.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -452064,6 +452064,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/600 {xref="Orphanet:425"} ! APOA1
 created_by: https://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100190
+name: gastric intestinal metaplasia
+def: "Metaplastic changes in the lining of the upper digestive tract." [https://my.clevelandclinic.org/health/diseases/22215-intestinal-metaplasia, PMID:29606921]
+synonym: "GIM" EXACT [PMID:29606921]
+synonym: "intestinal metaplasia" EXACT [NCIT:C8360, PMID:29606921]
+xref: NCIT:C8360 {source="MONDO:equivalentTo"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:452073]: I think you should use MONDO:relatedTo (as it is not equivalent)

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -452064,6 +452064,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
...
+is_a: MONDO:0004298 {source="https://orcid.org/0000-0001-7151-1615"} ! stomach disorder
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:452074]: Maybe there could be another parent: MONDO:0021074 - precancerous condition 
According to PMID:29606921:"Gastric intestinal metaplasia is a precancerous change of the mucosa of the stomach with intestinal epithelium, and is associated with an increased risk of dysplasia and cancer."

**Review by sabrinatoro — CHANGES_REQUESTED** [review]

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:452073]: good point

see this similar example

image

**Pull request merged**